modules/serverless-kn-func-run.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * serverless/cli_tools/kn-func-ref.adoc
+// * serverless/functions/serverless-functions-getting-started.adoc
+
+:_content-type: REFERENCE
+[id="serverless-kn-func-run_{context}"]
+= Running a function locally
+
+You can use the `kn func run` command to run a function locally in the current directory or in the directory specified by the `--path` flag:
+
+.Example command to run a function in the current directory
+[source,terminal]
+----
+$ kn func run
+----
+
+.Example command to run a function in a directory specified as a path
+[source,terminal]
+----
+$ kn func run --path=<directory_path>
+----
+
+If the function that you are running has never previously been built, the `kn func run` command builds the function before running it by default.
+
+You can also rebuild an existing image before running the function, by using the `--build` flag:
```

Comment:
```suggestion
You can also force a rebuild of an existing image before running the function, by using the `--build` flag:
```

Reply:
It doesn't really make sense to me why they would need to force a build if no files etc have changed? Shouldn't it just build automatically if there's a reason to?